engine/src/main/resources/assets/i18n/menu_de.lang

```diff
@@ -88,23 +104,34 @@
     "join-game-online": "Spiel beitreten",
     "join-server-action": "Spiel beitreten",
     "join-server-refresh": "Neuladen",
+    "join-server-requested": "angefragt",
     "join-server-title": "Server beitreten",
     "light-shafts": "Lichtstrahlen",
     "listed-servers": "Serverliste",
     "load-game": "Laden",
+    "menu-animations": "Animierte Menüs",
     "missing-name-message": "Name fehlt",
     "module-dependencies-empty": "Dieses Modul verfügt über keine Abhängigkeiten",
     "module-dependencies-exist": "Modulabhängigkeiten",
+    "module-filter": "Filter",
     "module-list": "Module",
     "module-name": "Name",
     "module-status": "Status",
+    "module-status-activated": "Aktiviert",
+    "module-status-activegameplay": "Aktives Gameplay",
+    "module-status-available": "Verfügbar",
+    "module-status-dependency": "Abhängigkeiten",
+    "module-status-error": "Inkompatible oder unaufgeklärte Abhängigkeiten ",
```

Comment:
In the context of dependencies, the word "unaufgelöst" is usually used instead of "unaufgeklärt"